cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -109,6 +109,18 @@ var (
 		RunE: forceNewEpochCmd,
 	}
 
+	setConfig = &cobra.Command{
+		Use:   "set-config",
```

Comment:
`set-config` is rather generic (we already have configs here), what about `set-network-config`?

Reply:
> rather generic
 
agree but i did new command as an opposite command to the `dump-config`. what do you think if there are two commands that work with the same config but names as `dump-config` and `set-network-config`? should it be renamed too?

Reply:
Yeah, I agree. Let's leave it `set-config` , given that there were no complaints for `dump-config`.